app/src/main/AndroidManifest.xml

```diff
@@ -26,9 +24,10 @@
         android:theme="@style/Theme.CultureQuest"
         android:usesCleartextTraffic="true"
         tools:targetApi="31">
-
+        <activity
+            android:name=".SettingsActivity"
+            android:exported="false" />
```

Comment:
Why make an activity of its own, did you consider having another fragment?

Reply:
I considered it but I think it's better to have a separate activity since I don't want the Navigation bar to be displayed.

Reply:
Ok! Just wanted to make sure :)